runtime/jcl/common/jclcinit.c

```diff
@@ -162,6 +162,11 @@ jint computeFullVersionString(J9JavaVM* vm)
 		}
 	}
 #endif
+
+#if !defined(OPENJ9_REPO)
+#define OPENJ9_REPO "OpenJ9   - "
+#endif /*OPENJ9_REPO*/
```

Comment:
nit: this should say `!OPENJ9_REPO`

Reply:
This isn't checking for a true/false value but if a the variable is a supplied value by the user. (eg.  Joe's OpenJ9).  Maybe I am not understanding here?

Reply:
Discussed in person...comment was for the closing endif comment.  I fixed that one and the others in the area that I was modifying.